worlds/ladx/ItemIconGuessing.py

```diff
@@ -0,0 +1,197 @@
+FORBIDDEN = [
+    "TRADING",
+    "ITEM",
+    "BAD",
+    "SINGLE",
+    "UPGRADE",
+    "BLUE",
+    "RED",
+    "NOTHING",
+    "MESSAGE",
+    "POWER",
+    "MAGIC",
+    "PEGASUS",
+    "TAIL",
+    "ANGLER",
+    "FACE",
+    "BIRD",
+    "SLIME",
+    "GOLD",
+    "MAX",
+    "UPGRADE",
+    "BOMBS",
```

Comment:
Why is this forbidden? We want Bombs/Arrows to show up as the proper thing

Reply:
"BOMBS" hits "MAX_BOMBS_UPGRADE", same for arrows. The BOMB and ARROW icons work better so I associate those instead.

Reply:
I guess my worry is making sure that "Power Bombs", "10 Bombs", etc get the right icon. Which BTW we should put a specific case for power bombs if you don't already have it.

Reply:
![P3_ladx_qBWLCehtTk651R1kccoSDQ-230921-152157](https://github.com/ArchipelagoMW/Archipelago/assets/26460970/68e05818-b0e0-4bcb-b906-64f324a68627)

Yeah it works, and power bombs as well with no special rule necessary. (I added `POWER` to the forbidden words list)

Reply:
![P3_ladx_S0gNPq-_T2GaOA1JCGLmRg-230921-160046](https://github.com/ArchipelagoMW/Archipelago/assets/26460970/b36ed173-1d42-4de3-974b-c5dc409aad83)